src/ui/qgsmeasurebase.ui

```diff
@@ -132,6 +131,26 @@
      </layout>
     </widget>
    </item>
+   <item row="5" column="0">
+    <widget class="QRadioButton" name="mCartesian">
+     <property name="text">
+      <string>Cartesian</string>
+     </property>
+     <property name="checked">
+      <bool>true</bool>
+     </property>
+    </widget>
+   </item>
+   <item row="5" column="2">
+    <widget class="QRadioButton" name="mProjected">
+     <property name="text">
+      <string>Projected</string>
```

Comment:
I'm not sure "Projected" is the best term here, because a user using a local projected CRS may confuse this with calculating cartesian measurements based on the local projection. Elsewhere we use "Cartesian"/"Ellipsoidal" to differentiate the two.

Reply:
I changed "Projected" to "Ellipsoidal"